docs/source/getting_started/getting_started.md

```diff
@@ -66,9 +66,36 @@ Each document has a unique identifier that can be used to refer to it in the sys
 
 The `snippet` field is used to inform the system about the content of the document; it is used as input to Xaynia to generate a mathematical representation of the document that we can use to match similar documents.
 
-For this reason, it is essential that the snippet clearly represents the content of the document. In this case, we took a few representative sentences from the document and used them as a snippet. Since the amount of data that Xaynia can analyze is limited, if it is not possible to provide a concise snippet, we can use the per-document option `summarize`; when enabled, the system will summarise the content of the document to create a snippet. Alternatively the `split` option can be used which will split the document into multiple snippets,
+For this reason, it is essential that the snippet clearly represents the content of the document. In this case, we took a few representative sentences from the document and used them as a snippet. Since the amount of data that Xaynia can analyze is limited, if it is not possible to provide a concise snippet, we can use the per-document option `summarize`; when enabled, the system will summarize the content of the document to create a snippet. Alternatively the `split` option can be used which will split the document into multiple snippets,
 this can be used to get better result on larger documents. More information on this can be found in the section  [split documents](#split-documents).
 
+For example the ingestion of:
+
+```bash
+curl -X POST "$URL/documents" \
+    --header "authorizationToken: $BACKOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "documents": [
+            {
+                "id": "xayn_efd5604c",
+                "snippet": "In a world [.. trimmed make the example readable ..]. We semantic [.. trimmed make the example readable ..].",
+                "split": true,
+            }
+        ]
+    }'
+
+```
+
+could split the text into two part:
+
+- `"In a world [.. trimmed make the example readable ..]."` with `"snippet_id": { document_id: "xayn_efd5604c",  sub_id: 802 }` and
+- `"We semantic [.. trimmed make the example readable ..]."` with `"snippet_id": { document_id: "xayn_efd5604c",  sub_id: 4 }`
+
+Be aware that `sub_id` is **not** guaranteed to have any specific ordering or structure, even through in practice
+it will most times look sequential incremental. Similar the natural language based splitting is not
+guaranteed to be deterministic. Especially over time with the algorithm being improved it will change.
```

Comment:
```suggestion
guaranteed to be deterministic. Especially over time, as we continue to improve the algorithm, the splitting may change.
```

Reply:
This is marked as resolved, but it doesn't seem to be applied.